miner/worker.go

```diff
@@ -1231,10 +1293,15 @@ func (w *worker) generateWork(genParams *generateParams) *newPayloadResult {
 	innerExecutionTimer.Update(core.DebugInnerExecutionDuration)
 
 	log.Debug("build payload statedb metrics", "parentHash", genParams.parentHash, "accountReads", common.PrettyDuration(work.state.AccountReads), "storageReads", common.PrettyDuration(work.state.StorageReads), "snapshotAccountReads", common.PrettyDuration(work.state.SnapshotAccountReads), "snapshotStorageReads", common.PrettyDuration(work.state.SnapshotStorageReads), "accountUpdates", common.PrettyDuration(work.state.AccountUpdates), "storageUpdates", common.PrettyDuration(work.state.StorageUpdates), "accountHashes", common.PrettyDuration(work.state.AccountHashes), "storageHashes", common.PrettyDuration(work.state.StorageHashes))
-
+	fees := big.NewInt(0)
+	if w.config.Mev.MevEnabled {
+		fees = work.profit
```

Comment:
```
	if w.config.Mev.MevEnabled {
		minerFee, _ := tx.EffectiveGasTip(env.header.BaseFee)
		env.profit.Add(env.profit, new(big.Int).Mul(new(big.Int).SetUint64(receipt.GasUsed), minerFee))
	}
```

What's the difference of the profit and totalFees?

Reply:
The calculation formulas of the profit and totalFees are the same. Totalfees is a loop to calculate all transactions, which is a relatively time-consuming operation. We now have two workers working, so my idea is to calculate it during the concurrent execution of each go routine. Calculate the result instead of waiting and then start two go routines to calculate the totalfees of the two workers. At the same time, regarding the atomicity of the bundle, set the profit to 0 to discard the worker. It is relatively simple to implement. Finally, we return the result if there is profit can be used directly without calculating totalfees again.

Reply:
Maybe something needs to be different here, totalfees is just to calculate miner fees, I think our profit needs to consider basefee and l1 fee, because basefee is our income and l1 fee is our cost.